tensorflow/contrib/distributions/python/ops/student_t.py

```diff
@@ -211,6 +211,14 @@ def _prob(self, x):
             (math_ops.sqrt(self.df) * math.sqrt(math.pi) * self.sigma) *
             math_ops.pow(1. + math_ops.square(y) / self.df, -(0.5 + half_df)))
 
+  def _log_cdf(self, x):
```

Comment:
I just realized you dont need this function. I forgot that we added base class code which does this automatically if subclass doesnt implement it.